README.md

```diff
@@ -39,7 +41,7 @@ Many models of Orange Pi hardware running [Armbian](https://www.armbian.com/) ar
     The installation script can also install many optional features such as distributed applications and network analysis tools that are useful but non-essential to run a node. You can use flags to selectively enable them, or use the following command to install all optional features:
 
     ```
-    $ wget https://raw.githubusercontent.com/tomeshnet/prototype-cjdns-pi/master/scripts/install && chmod +x install && WITH_MESH_POINT=true WITH_WIFI_AP=true WITH_CJDNS_IPTUNNEL=true WITH_IPFS=true WITH_PROMETHEUS_NODE_EXPORTER=true WITH_PROMETHEUS_SERVER=true WITH_GRAFANA=true WITH_H_DNS=true WITH_H_NTP=true WITH_FAKE_HWCLOCK=true WITH_EXTRA_TOOLS=true ./install
```

Comment:
I don't think anyone is using `WITH_FAKE_HWCLOCK` but curious why it was removed?

Reply:
I realized that it writes to the SD card every 5 mins, that wears the sd card down. So although I don't think it was actually wearing out the sd card in any significant manner (Prometheus is probably doing more writing then this is), looking at it as a whole it was doing more harm the good and didn't even fully solve the issue.